geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ShowMetricsInterceptor.java

```diff
@@ -102,4 +107,20 @@ private CommandResult getInvalidCategoryResult(Set<String> invalidCategories) {
     }
     return ResultBuilder.createUserErrorResult(sb.toString());
   }
+
+  @Override
+  public ResultModel postExecution(GfshParseResult parseResult, ResultModel resultModel,
+      Path tempFile) {
+    try {
+      for (Map.Entry<String, FileResultModel> entry : resultModel.getFiles().entrySet()) {
+        entry.getValue().saveFile(null);
```

Comment:
This is confusing. It looks like by running saveFile with null, you're saving it in the current directory. Perhaps you could create an overloaded method that doesn't take any parameters which would make it more readable.